File: test7/devowel_file.pl
#!/usr/bin/perl -w
$file = '';
open ($F,'<', $ARGV[0]);
while ($line = <$F> ) {
	$line =~ s/[aeiouAEIOU]//g;
	$file = $file.$line;
}
close $F;

open ($F,'>',$ARGV[0]);
print $F "$file";
close $F;

File: test7/devowel_file.py
#!/usr/bin/python3
import re,sys

file  = open (sys.argv[1],'r+')
newFile = ''
for line in file:
	newLine = re.sub("[aeiouAEIOU]",'',line)
	newFile = newFile + newLine
file.close ()

file  = open (sys.argv[1],'w')
file.write(newFile)	
file.close ()
